<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardRecharger</class>
 <widget class="QDialog" name="CardRecharger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CardRecharger</string>
  </property>
  <widget class="QGraphicsView" name="AdvertisementGraphics">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>721</width>
     <height>781</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="StatusLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>511</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Recharge30Button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>30元</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Recharge10Button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>10元</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Recharge80Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>80元</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Recharge20Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>20元</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Recharge15Button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>15元</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Recharge5Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>5元</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Recharge50Button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>50元</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Recharge100Button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>100元</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Recharge200Button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>200元</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="PayWay_Ali">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>490</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>支付宝</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="PayWay_WeiXin">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>560</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>微信支付</string>
   </property>
  </widget>
  <widget class="QLabel" name="PayWayLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择支付方式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="GreyRecordButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>511</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>灰记录处理</string>
   </property>
  </widget>
  <widget class="QLabel" name="QRcodeImageLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
